migration/migration_test.go

```diff
@@ -38,3 +38,24 @@ func TestConcurrentMigrations(t *testing.T) {
 	}
 	wg.Wait()
 }
+
+func TestMigrateWorkItemDescriptions(t *testing.T) {
+	resource.Require(t, resource.Database)
+	if err := configuration.Setup(""); err != nil {
+		panic(fmt.Errorf("Failed to setup the configuration: %s", err.Error()))
+	}
+	db, err := sql.Open("postgres", configuration.GetPostgresConfigString())
+	fmt.Println("Connecting to", configuration.GetPostgresConfigString())
+	if err != nil {
+		t.Fatalf("Cannot connect to DB: %s\n", err)
+	}
+	var tx *sql.Tx
+	tx, err = db.Begin()
+	defer tx.Commit()
+	if err != nil {
+		t.Fatalf("Cannot init tx: %s\n", err)
+	}
+	err = migrateWorkItemDescriptionsToMarkup()(tx)
+	assert.Nil(t, err)
```

Comment:
it makes sense, since I'll add more assertions afterwards. Otherwise, I think it was fine to end the test with `assert.Nil()`